pkg/systemd/quadlet/quadlet.go

```diff
@@ -101,6 +101,19 @@ const (
 	KeyYaml                  = "Yaml"
 )
 
+const (
+	SystemdSpecifierCacheDirectoryRoot                      = "%C"
+	SystemdSpecifierCredentialsDirectory                    = "%d"
+	SystemdSpecifierConfigurationDirectoryRoot              = "%E"
+	SystemdSpecifierUserHomeDirectory                       = "%h"
+	SystemdSpecifierLogDirectoryRoot                        = "%L"
+	SystemdSpecifierStateDirectoryRoot                      = "%S"
+	SystemdSpecifierRuntimeDirectoryRoot                    = "%t"
+	SystemdSpecifierDirectoryTemporaryFiles                 = "%T"
+	SystemdSpecifierDirectoryLargerPersistentTemporaryFiles = "%V"
+	SystemdSpecifierDirectoryFragment                       = "%Y"
```

Comment:
Some identifiers are missing (e.g, `%n`).  I would feel more comfortable to interpret anything starting with `%` with as an absolute path and document that in the Quadlet man page.

Otherwise, it's easy to miss existing identifiers and new ones in the feature.

Reply:
I was also thinking about which way would be better. I don't mind changing it

Reply:
Done